rust/src/nfs/nfs.rs

```diff
@@ -359,6 +350,19 @@ impl NFSState {
         let mut tx = NFSTransaction::new();
         self.tx_id += 1;
         tx.id = self.tx_id;
+        if self.transactions.len() > unsafe { NFS_MAX_TX } {
+            // set at least one another transaction to the drop state
+            for tx_old in &mut self.transactions {
+                if !tx_old.request_done || !tx_old.response_done {
+                    tx_old.request_done = true;
+                    tx_old.response_done = true;
+                    tx_old.is_file_closed = true;
+                    tx_old.tx_data.set_event(NFSEvent::TooManyTransactions as u8);
+                    self.events += 1;
```

Comment:
Q: what to we keep the events count for?

Reply:
It is indeed unused, added a new commit to remove it.

@jasonish is there some rust tool to detect this ?

Reply:
Do you mean unused variables?  Unread vars generate a warning by default.  However, this is a read and a write of the var, so probably not.  Maybe clippy can?

Reply:
Yes unused variables.
Oh, I see technically `+= 1` is a read and a write... But it is not so much of a read semantically...

Does clippy get run on CI somewhere ?